assets/stylesheets/host.css

```diff
@@ -9,4 +10,229 @@ body {
 .frame {
   height: 100%;
   width:  100%;
+  margin-top: 15px;
+}
+
+.modal-container {
+  animation: modalContainerFade ease 0.5s;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 1;
+}
+
+@keyframes modalContainerFade {
+  0% {
+    background-color: rgba(0, 0, 0, 0);
+  }
+  100% {
+    background-color: rgba(0, 0, 0, 0.5);
+   }
+}
+
+.modal {
+  animation: modalFade ease 0.5s;
```

Comment:
This animation could go to be a bit faster.

Reply:
Yep - changed it to 0.1s!